aliases/pckm.aliases.bash

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# Created by Jacob Hrbek github.com in 2019
+#  ---------------------------------------------------------------------------
+
+# Logic for root usage
+if [ -n "$EUID" ] && [ -x $(command -v sudo) ]; then
```

Comment:
This will always succeed without quoting.

```suggestion
if [ -n "$EUID" ] && [ -x "$(command -v sudo)" ]; then
```

Reply:
I'm again looking at this PR. There are additional several problems in this PR currently.

- `[ -n "$EUID" ]` tests if `EUID` is not an empty string, which will usually be always true. Here we should test that `EUID` is not an integer 0.
- Another condition for `[ -z "$OMB_USE_ROOT" ]` tests if `OMB_USE_ROOT` is empty. However, the effect should take effect only when it has the value `false` according to the modifications in README. Actually, the current implementation is counter-intuitive because oh-my-bash would *not* use the root privilege when any value is set to `OMB_USE_ROOT`.
- `$OMB_USE_ROOT` will fail with `set -u`.